bokehjs/src/coffee/common/canvas.coffee

```diff
@@ -43,7 +67,7 @@ class CanvasView extends ContinuumView
                           @ctx.mozBackingStorePixelRatio ||
                           @ctx.msBackingStorePixelRatio ||
                           @ctx.oBackingStorePixelRatio ||
-                          @ctx.backingStorePixelRatio || 1
+                          @ctx.backingStorePixelRatio || 1  # ak: wtf?
```

Comment:
I was asking what @almarklein meant behind this WTF (e.g. "using 1 as the default doesn't make sense, because ...") and what would be his solution.